target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -47,6 +47,10 @@ comfast,cf-e120a-v3)
 	ucidef_set_led_rssi "rssimediumhigh" "RSSIMEDIUMHIGH" "$boardname:green:rssimediumhigh" "wlan0" "51" "100"
 	ucidef_set_led_rssi "rssihigh" "RSSIHIGH" "$boardname:green:rssihigh" "wlan0" "76" "100"
 	;;
+comfast,cf-e314n-v2)
+	ucidef_set_led_netdev "wan" "WAN" "$boardname:white:wan" "eth0"
+	ucidef_set_led_netdev "lan" "lAN" "$boardname:white:lan" "eth1"
+	ucidef_set_led_wlan "wlan" "WLAN" "$boardname:white:wlan" "phy0tpt"
```

Comment:
@grosjo You can drop this line, as you've the LED defined already in your DTS with `linux,default-trigger = "phy0tpt";`